project-koku/koku
# [COST-2900] add date and source filtering to GCP HCS

## Jira Ticket

[COST-2900](https://issues.redhat.com/browse/COST-2900)

## Description

This change will alter our HCS queries for GCP to include filtering based on date and source.

## Testing

1. Checkout Branch
2. In your `.env` set `ENABLE_HCS_DEBUG=True` so HCS processing will always occur.
3. Restart Koku
4. Create and load some test customer data, such as: `make create-test-customer` followed by `make load-test-customer-data`
5. Check in minio for HCS GCP data and if you look at a file for a specific date, the entries should all be for the date the file is for whereas on main, it will contain _all_ the data

## Notes

...
